pages/platform/databases/postgresql_10_advanced_parameters_references/guide.en-gb.md

```diff
@@ -0,0 +1,115 @@
+| `pg.log_min_duration_statement` | long | Log statements that take more than this number of milliseconds to run, -1 disables |
+| `pg.log_temp_files` | long | Log statements for each temporary file created larger than this number of kilobytes, -1 disables |
+| `pg.max_files_per_process` | long | PostgreSQL maximum number of files that can be open per process |
+| `pg.max_locks_per_transaction` | long | PostgreSQL maximum locks per transaction |
+| `pg.max_logical_replication_workers` | long | PostgreSQL maximum logical replication workers (taken from the pool of max_parallel_workers) |
+| `pg.max_parallel_workers` | long | Sets the maximum number of workers that the system can support for parallel queries |
+| `pg.max_parallel_workers_per_gather` | long | Sets the maximum number of workers that can be started by a single Gather or Gather Merge node |
+| `pg.max_pred_locks_per_transaction` | long | PostgreSQL maximum predicate locks per transaction |
+| `pg.max_prepared_transactions` | long | PostgreSQL maximum prepared transactions |
+| `pg.max_replication_slots` | long | PostgreSQL maximum replication slots |
+| `pg.max_slot_wal_keep_size` | long | PostgreSQL maximum WAL size (MB) reserved for replication slots. Default is -1 (unlimited). wal_keep_size minimum WAL size setting takes precedence over this. |
+| `pg.max_stack_depth` | long | Maximum depth of the stack in bytes |
+| `pg.max_standby_archive_delay` | long | Max standby archive delay in milliseconds |
+| `pg.max_standby_streaming_delay` | long | Max standby streaming delay in milliseconds |
+| `pg.max_wal_senders` | long | PostgreSQL maximum WAL senders |
+| `pg.max_worker_processes` | long | Sets the maximum number of background processes that the system can support |
+| `pg.pg_partman_bgw.interval` | long | Sets the time interval to run pg_partman's scheduled tasks |
+| `pg.pg_partman_bgw.role` | string | Controls which role to use for pg_partman's scheduled background tasks. |
+| `pg.pg_stat_monitor.pgsm_enable_query_plan` | boolean | Enables or disables query plan monitoring |
+| `pg.pg_stat_monitor.pgsm_max_buckets` | long | Sets the maximum number of buckets  |
+| `pg.temp_file_limit` | long | PostgreSQL temporary file limit in KiB, -1 for unlimited |
+| `pg.timezone` | string | PostgreSQL service timezone |
+| `pg.track_activity_query_size` | long | Specifies the number of bytes reserved to track the currently executing command for each active session. |
+| `pg.track_commit_timestamp` | string | Record commit time of transactions. |
+| `pg.track_functions` | string | Enables tracking of function call counts and time used. |
+| `pg.track_io_timing` | string | Enables timing of database I/O calls. This parameter is off by default, because it will repeatedly query the operating system for the current time, which may cause significant overhead on some platforms. |
```

Comment:
```suggestion
| `pg.track_io_timing` | string | Enables timing of database I/O calls. This parameter is off by default, because it will repeatedly query the operating system for the current time, which may cause significant overheat on some platforms. |
```
Totally not-confident about this suggestion, overhead is a bit surprising.

Reply:
Keeping overhead